cmd/cli/proxy_configdump.go

```diff
@@ -0,0 +1,122 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"helm.sh/helm/v3/pkg/action"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+
+	"github.com/openservicemesh/osm/pkg/constants"
+)
+
+const dumpConfigDescription = `
+This command will dump the sidecar proxy configuration for the given pod.
+`
+
+const dumpConfigExample = `
+# Dump the proxy configuration for pod 'bookbuyer-5ccf77f46d-rc5mg' in the 'bookbuyer' namespace
+osm proxy dump-config bookbuyer-5ccf77f46d-rc5mg -n bookbuyer
+`
+
+type proxyDumpConfigCmd struct {
+	out        io.Writer
+	config     *rest.Config
+	clientSet  kubernetes.Interface
+	namespace  string
+	pod        string
+	localPort  uint16
+	sigintChan chan os.Signal
+}
+
+func newProxyDumpConfig(config *action.Configuration, out io.Writer) *cobra.Command {
+	dumpConfigCmd := &proxyDumpConfigCmd{
+		out:        out,
+		sigintChan: make(chan os.Signal, 1),
+	}
+
+	cmd := &cobra.Command{
+		Use:   "dump-config POD ...",
```

Comment:
Removing the ellipsis might make it more clear only one pod can be passed.
```suggestion
		Use:   "dump-config POD",
```

Reply:
The `...` is for the flags. Not required?